kaldb/src/main/java/com/slack/kaldb/elasticsearchApi/ElasticsearchApiService.java

```diff
@@ -61,6 +61,27 @@ public ElasticsearchApiService(KaldbQueryServiceBase searcher) {
     this.searcher = searcher;
   }
 
+  /** Returns metadata about the cluster */
+  @Get
+  @Path("/")
+  public HttpResponse clusterMetadata() {
```

Comment:
Due to https://github.com/grafana/grafana/commit/4d7c0904ef3455a2fc8db0ba813463c222d6c350